src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -19,31 +20,28 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU' " />
-
   <ItemGroup>
+    <Compile Include="System\Net\SecureChannel.cs" />
+    <Compile Include="System\Net\SecureProtocols\FixedSizeReader.cs" />
+    <Compile Include="System\Net\SecureProtocols\HelperAsyncResults.cs" />
+    <Compile Include="System\Net\SecureProtocols\SslState.cs" />
+    <Compile Include="System\Net\SecureProtocols\SslStreamInternal.cs" />
+    <Compile Include="System\Net\SecurityBuffer.cs" />
+    <Compile Include="System\Net\SecurityBufferType.cs" />
+    <Compile Include="System\Net\SslSessionsCache.cs" />
```

Comment:
I will rearrange files as previous like and just remove the "_" form them. Is that okie to do it?